### ISSUE TITLE
I think my settings are wrong. - 'idris.typecheck'

### ISSUE THREAD

**Issue by DUVELA** [zjhmale/vscode-idris#119]
I can a little understand English. I use a translator so it's ok.

OS : windows 10
C:\Program Files\Microsoft VS Code.
C:\Users\i\.vscode\extensions 
|ms-vscode.atom-keybindings-3.0.2
| vscode-idris-master
| zjhmale.idris-0.9.8

I can see that it works. 
"idris.add-clause", keybinding - Ctrl + Alt + A
"idris.case-split", keybinding - Ctrl + Alt + C
"idris.docs-for",   keybinding - Ctrl + Alt + D
"idris.make-lemma", keybinding - Ctrl + Alt + L
"idris.make-case",  keybinding - Ctrl + Alt + M
"idris.proof-search", keybinding - Ctrl + Alt + S
"idris.type-of",   keybinding - Ctrl + Alt + T

![default](https://user-images.githubusercontent.com/10277349/33617444-09cfd222-da23-11e7-8abe-76e0a9182544.jpg)
"idris.typecheck".. um.... hello???
"idris.typecheck" is no response.

what is a problem?
I wanna see that "file loaded successfully"  or error message.

**Comment by corazza** [comment]
Yeah it seems like the command isn't working: https://github.com/zjhmale/vscode-idris/issues/118

Which is too bad I really wanted to switch to VS Code, but I can't stand when trivial stuff doesn't work:)

**Comment by DUVELA** [comment]
Well ... I was forgetting vscode because I could not solve it. :)
